marlon/simulate.py

```diff
@@ -50,10 +51,11 @@ def run_simulation(iteration_count, agent_file):
 
         # If there is a jump in the reward for this step, record it for UI display.
         if done:
-            simulation.append(generate_graph_json(gym_env, iteration_count))
+            simulation.append(generate_graph_json(gym_env, iteration_count, current_score))
             break
         if reward != 0 or iteration == iteration_count-1:
-            simulation.append(generate_graph_json(gym_env, iteration+1))  
+            current_score += reward
+            simulation.append(generate_graph_json(gym_env, iteration+1, current_score))  
```

Comment:
If it's possible for done to be true and for a reward to be given at the same time the last reward will be missing. Move the `current_score += reward` before the ifs?